src/commands/utility/ServerInfoCommand.js

```diff
@@ -13,8 +13,10 @@ class ServerInfoCommand extends Command{
     async execute() {
         const guild = this.message.guild;
         
+        let owner = await this.bot.users.fetch(guild.ownerID);
+      
         let generic = '';
-        generic += `**Owner:** <@!${guild.ownerID}> \n`;
+        generic += `**Owner:** <@!${guild.ownerID}> (${owner.user.username}) \n`;
```

Comment:
you probably want to use .tag here (to include the discrim)

Reply:
I was lied to.
Another dev told me user.username would also include the discrim

Reply:
It hast been fixed